neo4j/assets/service_checks.json

```diff
@@ -1,17 +1 @@
-[
```

Comment:
why remove the service checks?

Reply:
bump, if you run this version of the integration `datadog-agent check neo4j` you will see a service check is submitted that looks like `neo4j.prometheus.health`. This is submitted in the base check you're inheriting: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/prometheus/mixins.py#L595

We should document that in service_checks.json


See docs for more info https://docs.datadoghq.com/developers/service_checks/